plugin/bundles/commodities_api/plugins/get_commodity_price_historical/plugin.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from aiohttp import ClientSession
+
+
+from bundle_dependency import *
+
+
+class GetCommodityPriceHistorical(PluginHandler):
+    async def execute(self, credentials: BundleCredentials, plugin_input: PluginInput) -> PluginOutput:
+        date: str = plugin_input.input_params.get("YYYY-MM-DD")
+        symbols: str = plugin_input.input_params.get("symbols").upper()
+        access_key: str = credentials.credentials.get("COMMODITIES_API_API_KEY", "")
+
+        url = f"https://commodities-api.com/api/{date}?access_key={access_key}&symbols={symbols}"
+
+        async with ClientSession() as session:
+            async with session.get(url) as response:
+                if response.status == 200:
+                    data = await response.json()
+                    data = data.get("data", {})
+                    date: str = data.get("date", None)
+                    base: str = data.get("base", None)
+
+                    rates: float = data.get("rates", None)
+                    inverted_rates = {symbols: 1 / rates[symbols]}
```

Comment:
double check if this endpoint supports multiple symbols, or strictly only one symbol